dev/dll/CommandingHelpers.cpp

```diff
@@ -110,16 +109,16 @@ winrt::IInspectable CommandingHelpers::WUXIconSourceToMUXIconSourceConverter::Co
     winrt::throw_hresult(E_NOTIMPL);
 }
 
-void CommandingHelpers::BindToLabelPropertyIfUnset(
+void CommandingHelpers::BindToIconSourcePropertyIfUnset(
```

Comment:
What's the purpose of these changes?

Reply:
Ah, ok. The two functions just switched place in the file.

---
In reply to: [302729511](https://github.com/microsoft/microsoft-ui-xaml/pull/956#discussion_r302729511) [](ancestors = 302729511)